webpack.config.js

```diff
@@ -131,6 +131,8 @@ module.exports = {
       APP_ID: JSON.stringify(process.env.APP_ID),
       DEBUG_LIFF: process.env.DEBUG_LIFF,
       COFACTS_API_URL: JSON.stringify(process.env.API_URL),
+      'process.env.SITE_URL': JSON.stringify(process.env.SITE_URL),
+      'process.env.LOCALE': JSON.stringify(process.env.LOCALE),
```

Comment:
Forgot to add `LOCALE` to `.env.sample` in this [PR](https://github.com/cofacts/rumors-line-bot/pull/146).

Reply:
Fixed, thanks